src/utils/rpcQueries/viewFunction.ts

```diff
@@ -30,7 +30,7 @@ export default async function viewFunction({
     request_type: 'call_function',
     account_id: contractId,
     method_name: method,
-    args_base64: Buffer.from(JSON.stringify(args)).toString('base64'),
+    args_base64: btoa(JSON.stringify(args)),
```

Comment:
This will fail in Node.js context as `btoa` is only in the browser. 

Suggestion: you could wrap it in a try/catch and fallback to `Buffer.toString('base64)`.